.github/workflows/ci.yml

```diff
@@ -42,7 +42,7 @@ jobs:
           echo "VITE_BACKEND_API_BASE_URL=http://localhost:8000/" >> env/frontend.env
           docker compose run ci-backend
       - name: Run mypy checks
-        run: python -m mypy ./backend
+        run: cd backend && python -m mypy . && cd ..
```

Comment:
Is that `cd ..` needed?

Also, if the mypy has errors, this won't go up a directory and try to run everything else from inside `backend`

Reply:
Yea it's needed to switch dirs to pick up my config. Doesn't really matter that it will not run cd .. because if it fails, the pipeline aborts

Reply:
ahhh, understandable